modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java

```diff
@@ -209,9 +209,13 @@ else if (cmd.equals("cleanUpOrphanedPortletPreferences")) {
 		else if (cmd.startsWith("convertProcess.")) {
 			redirect = _convertProcess(actionRequest, actionResponse, cmd);
 		}
-		else if (cmd.equals("dlPreviews")) {
+		else if (cmd.equals("dlDeleteFiles")) {
```

Comment:
Can we use a more specific name, like `dlDeletePreviews`? This will also be consistent with the other one (`dlGeneratePDFPreviews`).

Reply:
done